File: frontend/src/app/models/ReviewerQuestionnaire.ts
export class ReviewerQuestionnaire{
  
    idUser?: number
    idQuestionnaire?: number
    isDeleted?: boolean

    constructor({idUser, idQuestionnaire, isDeleted}:{idUser?: number, idQuestionnaire?: number, isDeleted?: boolean}) {
        this.idUser=idUser;
        this.idQuestionnaire=idQuestionnaire;
        this.isDeleted=isDeleted;
    }

}

File: backend/CoreCuestionariosOIJ/src/CuestionariosEntidades/Models/ReviewerQuestionnaire.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CuestionariosEntidades.Models
{
    public class ReviewerQuestionnaire
    {
        public int? IdUser { get; set; }
        public int? IdQuestionnaire { get; set; }
        public bool? IsDeleted { get; set; }

        // EF Relation
        public Questionnaire? Questionnaire { get; set; }
    }
}
